airbyte-scheduler/client/src/main/java/io/airbyte/scheduler/client/SynchronousResponse.java

```diff
@@ -22,12 +22,13 @@ public static <T> SynchronousResponse<T> success(final T output, final Synchrono
     return new SynchronousResponse<>(output, metadata);
   }
 
-  public static <T> SynchronousResponse<T> fromTemporalResponse(final TemporalResponse<T> temporalResponse,
-                                                                final UUID id,
-                                                                final ConfigType configType,
-                                                                final UUID configId,
-                                                                final long createdAt,
-                                                                final long endedAt) {
+  public static <T, U> SynchronousResponse<T> fromTemporalResponse(final TemporalResponse<U> temporalResponse,
```

Comment:
Because I wanted to keep the SynchronousResponse the same instead of the more generic type, the output is mapped and provided directly as an argument, so the TemporalResponse type and SynchronousResponse types can be different. 

For example, for the discover job we have `TemporalResponse<ConnectorJobOutput>` and `SynchronousResponse<AirbyteCatalog>`